clang/include/clang/AST/ASTContext.h

```diff
@@ -200,7 +200,7 @@ class ASTContext : public RefCountedBase<ASTContext> {
   mutable llvm::ContextualFoldingSet<ConstantArrayType, ASTContext &>
       ConstantArrayTypes;
   mutable llvm::FoldingSet<IncompleteArrayType> IncompleteArrayTypes;
-  mutable std::vector<VariableArrayType*> VariableArrayTypes;
+  mutable std::vector<VariableArrayType *> VariableArrayTypes;
```

Comment:
Did you run clang-format on this entire file? LLVM's coding standards don't allow that. You'll need to undo the formatting changes that aren't related to your patch.

Reply:
That was `git clang-format` in action, yes.

(Why is ASTContext.h/cpp not clang-format clean?)